### PR TITLE
Better config errors

### PR DESCRIPTION
**Pull request by camertron**
@lumoslabs/platform

### CODE REVIEW

File: lib/txgh/config/providers/git_provider.rb [hunk 1]
```diff
@@ -36,7 +36,7 @@ def config
         def download
           github_repo.api.download(github_repo.name, payload, ref)
         rescue Octokit::NotFound
-          raise ConfigNotFoundError, "Config file #{payload} not found in #{ref}"
+          raise GitConfigNotFoundError, "Config file #{payload} not found in #{ref}"
```
Comment (niwen) [inline lib/txgh/config/providers/git_provider.rb]: Does this not need `Txgh::`? Seems like it's defined in the same place as the other errors.

File: spec/config/key_manager_spec.rb [hunk 2]
```diff
@@ -18,6 +18,23 @@
...
+    end
+
+    it "raises an error if the scheme can't be recognized" do
+      # remove the scheme
+      allow(Txgh::Config::KeyManager).to(
+        receive(:raw_config).and_return(YAML.dump(base_config))
+      )
+
+      expect { KeyManager.config_from_project(project_name) }.to(
+        raise_error(InvalidProviderError)
```
Comment (niwen) [inline spec/config/key_manager_spec.rb]: Same - does `Txgh::` need to be added here and in line 65?

Comment (camertron) [inline spec/config/key_manager_spec.rb]: Good catch!

Comment (camertron) [inline spec/config/key_manager_spec.rb]: Oh actually... the tests usually include `Txgh` at the top. But I can make this more consistent.

**Comment by niwen** [comment]
👍

**Pull request merged**